src/app/contextBroker/contextBroker.cpp

```diff
@@ -1738,11 +1739,20 @@ int main(int argC, char* argV[])
   orionInit(orionExit, ORION_VERSION, policy, statCounters, statSemWait, statTiming, statNotifQueue, strictIdv1);
   mongoInit(dbHost, rplSet, dbName, user, pwd, mtenant, dbTimeout, writeConcern, dbPoolSize, statSemWait);
   contextBrokerInit(dbName, mtenant);
-  curl_global_init(CURL_GLOBAL_NOTHING);
   alarmMgr.init(relogAlarms);
   metricsMgr.init(!disableMetrics, statSemWait);
   logSummaryInit(&lsPeriod);
 
+  // According to http://stackoverflow.com/questions/28048885/initializing-ssl-and-libcurl-and-getting-out-of-memory/37295100,
+  // openSSL library needs to be initialized with SSL_library_init() before any use of it by other the other libraries
```

Comment:
before any use of it by other the other libraries ... 

before any use of it by any other library?